test/src/820-https_s3/main

```diff
@@ -8,24 +8,21 @@ cvmfs_test_name="HTTPS S3 implementation"
 cvmfs_test_autofs_on_startup=false
 cvmfs_test_suites="quick"
 
-set -x
-
 SETTING_DIR="/tmp/cvmfs_testing_out"
-MINIO_PID=""
 
 clean_up() {
     echo "Cleaning up"
     # stop minio
     sudo kill -9 $(cat $SETTING_DIR/minio_pid)
-    
+
     # remove the repository
-    sudo cvmfs_server rmfs -f https-test.cvmfs.ch    
-    
+    sudo cvmfs_server rmfs -f https-test.cvmfs.ch
+
     # remove all the configuration files
     sudo rm -rf "/tmp/cvmfs_testing_out"
```

Comment:
If the files are created in the working directory of the test, the test driver takes care of removal on test success.